packages/taquito-local-forging/src/taquito-local-forging.ts

```diff
@@ -25,10 +26,10 @@ export function getCodec(codec: CODEC) {
   };
 }
 
-export { VERSION } from './version';
-
 export class LocalForger implements Forger {
-  private codec = getCodec(CODEC.MANAGER);
+  constructor(public readonly protocolHash = ProtocolsHash.PtHangz2) {}
```

Comment:
will this be something we'd change in the future in accordance to protocol updates?

Reply:
Yes, we will need to update the default value to match new protocol hash

Reply:
maybe passing a config that is named `currentProtocol` or something like that would be more descriptive, and instead we can just modify the value of `currentProtocol` and not pass them directly to the constructor

Reply:
I thought about it a little more, and I think that I will change my precedent response. The schema of endorsement is different in hangzhou and ithaca. So at the moment, we need to know which protocol is used to forge this kind of operation properly. The distinction is needed until the migration from hangzhou to ithaca on mainnet. After that, it won't really make sense to forge using the old schema and we should remove the distinction to keep the code cleaner.